src/redux/slices/meetingSlice.js

```diff
@@ -146,9 +146,12 @@ export const addMessage = createAsyncThunk(
 
 export const addRating = createAsyncThunk(
   'meeting/addRating',
-  async ({ token, meetingId, newRating }, { rejectWithValue }) => {
+  async ({ meetingId, newRating }, { rejectWithValue }) => {
     try {
-      const { data } = axios.post(
+      const token = localStorage.getItem('token');
+      if (!token) throw Error('Must be logged in to leave a rating');
```

Comment:
really not sure: does it need to be *new* Error()? if it works as is then i'm happy